projects/js-packages/components/components/contextual-upgrade-trigger/index.tsx

```diff
@@ -20,8 +21,19 @@ const ContextualUpgradeTrigger: React.FC< CutBaseProps > = ( {
 	return (
 		<div className={ classnames( styles.cut, className ) }>
 			<div>
-				<Text>{ description }</Text>
 				<Tag { ...tagProps }>
+					<Text>
+						{ description }{ ' ' }
+						{ tooltipText && (
+							<span className="tooltip">
+								<Tooltip position="middle center" text={ tooltipText }>
```

Comment:
I think we should not put a Tooltip inside here, because it has a `div` which should not be a descendant of `p` or `span`. If you hover on the icon and check the console, React shows a warning.

Reply:
I got rid of the parent <span> and replaced it with <>, do you think it will work?